docs/guides/designing-the-delta-format.md

```diff
@@ -16,7 +16,7 @@ If you are looking for a reference on *what* Deltas are, the [Delta documentatio
 
 ## Plain Text
 
-Let's start at the basics with just plain text. There already is a ubiquitous format to store plain text: the string. Now if we want to build upon this and describe formatted text, such as when a range is bold, we need to add additional information.
+Let's start at the basics with just plain text. There already is an ubiquitous format to store plain text: the string. Now if we want to build upon this and describe formatted text, such as when a range is bold, we need to add additional information.
```

Comment:
Technically 'an' ubiquitous is not wrong, but 'a' is far more common especially nowadays. http://english.stackexchange.com/questions/280921/a-or-an-ubiquitous